### ISSUE TITLE
When AutoPopDelay is the custom value, the keyboard toolTip for listview control is disappearing after pressing Ctrl

### ISSUE THREAD

**Issue by Amy-Li03** [dotnet/winforms#5852]
* OS: Win11

* .NET Core Version: 
.NET SDK 6.0.100-rtm.21473.61

* Have you experienced this same bug with .NET Framework?: 
No

**Minimal repro:**
1. Create a Winforms .Net Core app with a button, a ListView and a toolTip added
2. Set tooltip's AutoPopDelay value to: 9000
3. Set the ToolTip on toolTip1 of ListView is "this is a ListView"
3. Set the ToolTip on toolTip1 of Button is "this is a Button"
4. Build and run this project
5. Press Tab to show keyboard tooltip of ListView control ---  Now keyboard tooltip is appearing
6. Press Ctrl for listView control with keyboard tooltip to see if it is disappearing

**Actual behavior:**
The keyboard toolTip for listView control is disappearing
![Repro](https://user-images.githubusercontent.com/62929087/134625474-803217c3-6959-4c04-93e2-865aa3640933.gif)

**Expected behavior:**
The keyboard toolTip for listView control should not disappear

**More info:**
1. Pressing ESC keyboard doesn't repro this issue, keyboard toolTip for listview control is still appearing
2. Other controls cannot repro this issue
3. This issue only reproes on win11 OS, doesn't repro on win10 OS

**Comment by Tanya-Solyanik** [comment]
@Amy-Li03 - is this specific to the  ListView control? I suspect that this is a native control issue.

**Comment by Amy-Li03** [comment]
@Tanya-Solyanik yes, this issue is specific to ListView control, other controls don't repro this issue.

**Comment by NikitaSemenovAkvelon** [comment]
It doesn't reproduce currently. Maybe one of updates for Win11 fixed the bug, as you say previously it depends on OS.
![0FRHKvLqGj](https://user-images.githubusercontent.com/109065597/182091684-776e0d83-9f7a-4bc4-b07d-b2c666a89699.gif)

@Amy-Li03 could you test it once again, please?

**Comment by Amy-Li03** [comment]
Hi @NikitaSemenovAkvelon , verified this issue on OS: Win 11 22H2(OS Build 25153.1000) and .NET latest build: .NET SDK 7.0.100-rc.1.22401.1, issue is still reproducing, please see following gif.
![Repro](https://user-images.githubusercontent.com/62929087/182318497-a15a4ea3-aa17-4198-93e5-db1c672f2fd8.gif)

**Comment by Tanya-Solyanik** [comment]
@merriemcgaw  - this is likely to be a windows issue, as ListView tooltips are special-cased for this control. I suggest "won't fix". Winforms is not special-casing ListView right now, and shouldn't in the future.
